assets/chat/js/messages/ChatUserMessage.js

```diff
@@ -126,7 +126,9 @@ export default class ChatUserMessage extends ChatMessage {
     this.ui.classList.toggle('msg-me', isSlashMe);
     const ctrl = this.ui.querySelector('.ctrl');
     if (ctrl) {
-      ctrl.textContent = this.slashme || this.continued ? '' : ': ';
+      ctrl.textContent = ': ';
+      if (this.target) ctrl.textContent = ' whispered: ';
+      else if (this.slashme || this.continued) ctrl.textContent = '';
```

Comment:
When setting `slashme`, we only care to update the control if the message isn't a whisper. If it is a whisper, the control was already set to the correct value when the message was first built. Would you mind updating this logic to reflect that?